client-js/app/elements/labrad-plot.ts

```diff
@@ -133,6 +138,9 @@ export class Plot extends polymer.Base {
   private y0: number = null;
   private dx0: number = -1;
   private dy0: number = -1;
+  private displayTraces: Array<number>;
+  private allOrNone: boolean = true;
+  private oneIndep: boolean;
```

Comment:
I'd prefer to call this `is1D` or something like that. Can also have an `is2D` property for 2D data sets. There are other things we may need to change in the UI based on the dimensionality, so these are useful properties to have.